src/history/RepositoryViewDelegate.cpp

```diff
@@ -515,26 +515,25 @@ void RepositoryViewDelegate::paintTagBranch(QPainter *painter, QStyleOptionViewI
          o.font.setBold(isCurrentSpot);
 
          const auto nameToDisplay = showMinimal ? QString(". . .") : *mapIt;
-         QFontMetrics fm(o.font);
+         const QFontMetrics fm(o.font);
```

Comment:
Since you're modifying this line, I think it would be even better to merge this with line below:
`const auto textBoundingRect = QFontMetrics(o.font).boundingRect(nameToDisplay);`